learn.md

```diff
@@ -106,7 +106,7 @@ redirect_from:
 <h2 id="get-started">Get started</h2>
 <div class="row">
 <div class="col-lg-6 col-md-6 col-sm-12 card" >
-  <a href="/learn/install-ballerina/">
+  <a href="/learn/install-ballerina/set-up-ballerina/">
```

Comment:
Why  are we changing this on this PR?

Reply:
Added this as it is a minor change for a separate PR.

Reply:
Send it as a seperate PR

Reply:
Removed.